fastcrypto/src/bls12381/mod.rs

```diff
@@ -273,7 +273,7 @@ impl VerifyingKey for BLS12381PublicKey {
 }
 
 fn get_128bit_scalar<Rng: AllowedRng>(rng: &mut Rng) -> blst_scalar {
-    assert!(BLS_BATCH_RANDOM_SCALAR_LENGTH == 128);
+    assert!(BLS_BATCH_RANDOM_SCALAR_LENGTH <= 128);
```

Comment:
I'd use some const value that we can control in the future

Reply:
do we even need that assertion? IMO it's obsolete

Reply:
changed to debug assert